src/net.cpp

```diff
@@ -2046,7 +2054,7 @@ void CConnman::OpenNetworkConnection(const CAddress& addrConnect, bool fCountFai
     }
     if (!pszDest) {
         if (IsLocal(addrConnect) ||
-            FindNode(static_cast<CNetAddr>(addrConnect)) || (m_banman && m_banman->IsBanned(addrConnect)) ||
+            FindNode(static_cast<CNetAddr>(addrConnect)) || (m_banman && (m_banman->IsDiscouraged(addrConnect) || m_banman->IsBanned(addrConnect))) ||
```

Comment:
This would be easier to read if `(m_banman .... )` conditional was on its own line, so it's easier to see that the `||` within that conditional is not at the top level.

Reply:
Done.